qdata/parser.py

```diff
@@ -217,13 +286,42 @@ def qop(self, *args):
         if args[0] == "reset":
             # reset <argument>;
             wires = args[1].list
-        elif args[0] == "measure":
+            return Op(Ops.RESET, name=args[0], params=[], wires=wires)
+
+        if args[0] == "measure":
             wires = args[1:]
             return Measure(wires=wires)
 
-        return Op(name=args[0], params=[], wires=wires)
+        # TODO: Check the operator kind this 'catch-all' return
+        # actually refers to. Perhaps this return statement isn't even covered?
+        return Op(Ops.GATE, name=args[0], params=[], wires=wires)
 
     def uop(self, *args):
+        """A unitary operation.
+
+        This can consist of:
+
+        * Intrinsic quantum operations (U, CX)
```

Comment:
"Built-in supported quantum operations"?

Reply:
Intrinsic seems more apt? https://en.wikipedia.org/wiki/Intrinsic_function